src/pages/home/home.ts

```diff
@@ -462,41 +465,53 @@ export class HomePage {
       });
   }
 
+  //Alert control for map navigation must be created before clickFunction because of scoping issues.
+  prepareAlertCtrl(launchNav, currentPosition, selectedLocation) {
+    let alertCtrl = this.alertCtrl.create({
+      title: 'Redirect to navigator app',
```

Comment:
I don't really like this title... maybe something like 'Betwixt' for the default title of the alert will do.

Reply:
Done!